CodeStarterCamp_Week1/Stickbody.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Stickbody.swift
+//  CodeStarProject
+//
+//  Created by 유지완 on 2023/08/24.
+//
+
+import Foundation
+
+
+// 막대길이 4
+func createBodyStick4(stickbody : Int = 4){
```

Comment:
```suggestion
func createBodyStick4(stickBody: Int = 4){
```
카멜케이스 맞춰주시고 파라미터 네임뒤에 콜론은 붙여주도록 하세요